cea/guide_about_files_generated.md

```diff
@@ -0,0 +1,303 @@
+Guide
+=======
+**Author : Sreepathi Bhargava Krishna**
```

Comment:
I'd prefer this file to be part of the `docs/` structure. ReStructuredText also has some better mechanisms for meta-data (like author and date). The intent doesn't need to be marked, you can just use the first paragraph for that. Or an Abstract section. Also, why whole lines in bold? Maybe we can sit and talk about document structure someday.